recipes/dparse/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "dparse" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00a5fdfa900629e5159bf3600d44905b333f4059a3366f28e0dbd13eeab17b19
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - packaging
+    - python
+    - pyyaml
+    - six
+
+test:
+  imports:
+    - dparse
+  requires:
+    - pytest
```

Comment:
Import tests shouldn't have additional requires, this can be omitted. `test->requires` are the requirements for running the  `package tests`, not the test requirements of the package.

Reply:
Thanks for the feedback. I've removed as commented above.
The template was generated with `conda skeleton` so I suppose it is eagerly parsing the `test_requires`  of `setup.py`

Reply:
Yes, that is correct.

Reply:
But import tests are fine on conda-forge ;)